### ISSUE TITLE
<14/12/23><NY><Android><Reallocation><Validate if reallocation is not happening if driver cancel ride after 30 mits as bap.searchrequest expires after 30 mits><parent id 1980> 

### ISSUE THREAD

**Issue by 123ajay1994** [nammayatri/nammayatri#4753]
#1980

**Describe the bug**
Reallocation is happening via UI if driver cancel ride after 30 mits even if  bap.searchrequest expires after 30 mits.

**To Reproduce**
[prerequisite: make other drivers available in pool] 
- Book a ride
- Cancel the from driver side after 30 mits
- Validate if no re-allocation is happening.

**Expected behavior**
User side screen should be at Home Screen as SearchRequest/estimate Valid till 30 mits only.

**Actual behavior**
User side screen navigated to “Ride Search” Screen as in Reallocation but no new Driver received any Ride Request. But in same secession, when Tip is applied, RideRequest has been received to blocked driver as well. Seems creation of new estimate after applying the tip.

**Screenshots**
https://drive.google.com/file/d/1OmM9OKeUgIcc-XxIeOYEWMGDl1yT9f_I/view?usp=sharing

**App (please complete the following information):**

**Additional context**

**Comment by ashishsingh101** [comment]
@123ajay1994 can you update this ?

**Comment by 123ajay1994** [comment]
yes @ashishsingh101 now the time validity is of 5 mit only. So if bpp driver cancel the ride within 5 mits, than only re-allocation will happen.
cc: @prashant601 @SrinivasNaikB this config is in master env.

**Comment by 123ajay1994** [comment]
hi @prash010 validity is still 30 mits only. I have re-confirmed with DB. So re-allocation is happening when it ride cancellation within 30 mits.
cc: @ashishsingh101 

![Image](https://github.com/nammayatri/nammayatri/assets/128033831/94e38e48-6676-4f80-90d5-61e9beffb3f1)
